source/StigData/Processed/IISSite-10.0-2.4.xml

```diff
@@ -816,8 +816,8 @@ If the system must require "Request Limit" to be set to "0", it is documented an
       <IsNullOrEmpty>False</IsNullOrEmpty>
       <Key>logEventOnRecycle</Key>
       <LegacyId>V-100271</LegacyId>
-      <OrganizationValueRequired>False</OrganizationValueRequired>
-      <OrganizationValueTestString />
+      <OrganizationValueRequired>True</OrganizationValueRequired>
+      <OrganizationValueTestString>'{0}' 'Value must contain Time and Schedule but can contain Requests, Memory, IsapiUnhealthy, OnDemand, ConfigChange, PrivateMemory'</OrganizationValueTestString>
```

Comment:
I think the default value should still be "Time,Schedule", based on the wording of the STIG.. having all settings could be too strict of a setting. Org setting is definitely the way to go though, so users could add additional options

Reply:
Those are all set by default. If we don't include those, they will be off my default.